mobile/lib/services/auth.service.dart

```diff
@@ -46,6 +56,28 @@ class AuthService {
     return validUrl;
   }
 
+  Future<bool> validateAuxilaryServerUrl(String url) async {
+    final httpclient = HttpClient();
+    final accessToken = _authRepository.getAccessToken();
+    bool isValid = false;
+
+    try {
+      final uri = Uri.parse('$url/users/me');
```

Comment:
Is it okay to hardcode the endpoint like this?

Reply:
For this case, yes, because we are not using the OpenAPI instance to make the request. This is for checking if the endpoint you enter is valid